### PR TITLE
Refresh orbs intro

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Content refresh of orbs intro page

# Reasons
Needed a refresh

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/orb-intro.md [hunk 1]
```diff
@@ -12,40 +12,103 @@ contentTags:
   - Server v3.x
 ---
 
-* TOC
-{:toc}
+Use orbs to:
```
Comment (devinmarieb) [inline jekyll/_cci2/orb-intro.md:12]: Would this list not go under the introduction section? It seems odd to have an intro before an intro.

File: jekyll/_cci2/orb-intro.md [hunk 2]
```diff
@@ -12,40 +12,103 @@ contentTags:
...
 
 ## How orbs work
+{: #how-orbs-work }
+
+Orbs are made up of reusable configuration elements, for example, [jobs]({{site.baseurl}}/reusing-config/#authoring-parameterized-jobs), [commands]({{site.baseurl}}/reusing-config/#authoring-reusable-commands), [executors]({{site.baseurl}}/reusing-config/#executor).
+
+### Usage
+{: #usage }
+
+An orb is identified by its _slug_ which contains the _namespace_ and _orb name_. A namespace is a unique identifier referring to the organization authoring a set of orbs. The orb name will be followed by an `@` symbol and a [semantic version]({{site.baseurl}}/orb-concepts/#semantic-versioning) string, identifying which version of the orb is being used.
```
Comment (devinmarieb) [inline jekyll/_cci2/orb-intro.md]:
```suggestion
An orb is identified by its _slug_, which contains the _namespace_ and _orb name_. A namespace is a unique identifier referring to the organization authoring a set of orbs. The orb name will be followed by an `@` symbol and a [semantic version]({{site.baseurl}}/orb-concepts/#semantic-versioning) string, identifying which version of the orb is being used.
```

File: jekyll/_cci2/orb-intro.md [hunk 3]
```diff
@@ -12,40 +12,103 @@ contentTags:
...
+
+Example orb slug: `<namespace>/<orb-name>@1.2.3`
+
+Each orb within the [registry](https://circleci.com/developer/orbs) provides a [quickstart guide](/developer/orbs/orb/circleci/node#quick-start), which contains a sample code snippet for importing that specific orb, with its most recent version, into your `.circleci/config.yml`.
+
+The example below shows how to import any orb into your CircleCI configuration file, there are two tabs to show both a generic layout for importing any orb, and a specific example of importing the Node.JS orb:
```
Comment (devinmarieb) [inline jekyll/_cci2/orb-intro.md:40]:
```suggestion
The example below shows how to import any orb into your CircleCI configuration file. There are two tabs to show both a generic layout for importing any orb, and a specific example of importing the Node.JS orb:
```

**Comment by devinmarieb** [comment]
@rosieyohannan some minor things that you are free to ignore, and few things that I think need a look - the intro section of the page and an external link format.

**Review by devinmarieb — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/orb-intro.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/orb-intro.md:12]: https://circleci.atlassian.net/wiki/spaces/PEX/pages/6679953646/WIP+Content+templates#Conceptual we added a section pre-intro to highlight use cases after feedback that people want to know before reading whether content is relevant for them. Happy to talk this over though!

File: jekyll/_cci2/orb-intro.md [hunk 4]
```diff
@@ -12,40 +12,103 @@ contentTags:
...
+Orbs are made up of reusable configuration elements, for example, [jobs]({{site.baseurl}}/reusing-config/#authoring-parameterized-jobs), [commands]({{site.baseurl}}/reusing-config/#authoring-reusable-commands), [executors]({{site.baseurl}}/reusing-config/#executor).
+
+### Usage
+{: #usage }
+
+An orb is identified by its _slug_ which contains the _namespace_ and _orb name_. A namespace is a unique identifier referring to the organization authoring a set of orbs. The orb name will be followed by an `@` symbol and a [semantic version]({{site.baseurl}}/orb-concepts/#semantic-versioning) string, identifying which version of the orb is being used.
+
+Example orb slug: `<namespace>/<orb-name>@1.2.3`
+
+Each orb within the [registry](https://circleci.com/developer/orbs) provides a [quickstart guide](/developer/orbs/orb/circleci/node#quick-start), which contains a sample code snippet for importing that specific orb, with its most recent version, into your `.circleci/config.yml`.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/orb-intro.md:38]: OOops no that second link doesn't work. Thanks!

File: jekyll/_cci2/orb-intro.md [hunk 5]
```diff
@@ -1,45 +1,105 @@
...
+* Follow our [Node.js project quickstart guide](/docs/language-javascript/).
+* Follow our [Python project quickstart guide](/docs/language-python/).
+* Set up notifications using the [Slack orb](/docs/slack-orb-tutorial/).
+
+## Use an orb
+{: #use-an-orb }
+
+An orb is identified by its _slug_ which contains the _namespace_, and _orb name_. A namespace is a unique identifier referring to the organization authoring a set of orbs. The orb name will be followed by an `@` symbol and a [semantic version]({{site.baseurl}}/orb-concepts/#semantic-versioning) string, identifying which version of the orb is being used. For example: `<namespace>/<orb-name>@1.2.3`. 
+
+Each orb within the [registry](https://circleci.com/developer/orbs) provides a [quickstart guide](/developer/orbs/orb/circleci/node#quick-start), which contains a sample code snippet for importing that specific orb, with its most recent version, into your `.circleci/config.yml`.
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-intro.md:38]:
```suggestion
Each orb within the [registry](https://circleci.com/developer/orbs) provides a [quickstart guide](https://circleci.com/developer/orbs/orb/circleci/node#quick-start), which contains a sample code snippet for importing that specific orb, with its most recent version, into your `.circleci/config.yml`.
```

File: jekyll/_cci2/orb-intro.md [hunk 6]
```diff
@@ -85,81 +142,40 @@ The [Orb Registry](https://circleci.com/developer/orbs) is an open repository of
 
 ![Orb Registry]({{site.baseurl}}/assets/img/docs/orbs-registry.png)
 
+### Orb designations
+{: orb-designation }
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-intro.md:146]:
```suggestion
{: #orb-designation }
```

**Review by devinmarieb — APPROVED** [review]

**Pull request merged**